openapi/openapi.yaml

```
@@ -1529,7 +1529,7 @@ components:
         Names of the Python Packages (name, version, index) for the given import package name
       schema:
         type: string
-        example: sklearn.cluster.MiniBatchKMeans
+        example: sklearn
```

However, lets use the this
```suggestion
        example: sklearn.neural_network
```

Verified on Prod: https://khemenu.thoth-station.ninja/api/v1/python/imports?import_name=sklearn.neural_network

---

Thank you! Adding the change :+1: